src/components/LoginModal/index.tsx

```diff
@@ -20,8 +22,9 @@ interface LoginModal extends ModalProps {
 export default function LoginModal({ page, setPage, ...props }: LoginModal) {
   const { setUser } = useAuth()
   const { t } = useTranslation()
+  const [signupEmail, setSignupEmail] = useState<string>('')
```

Comment:
This state is already tracked by the AuthProvider within the `user` object. The user object can be imported on line 23: `const { setUser, user } = useAuth()`